tests/profile_test.py

```diff
@@ -0,0 +1,103 @@
+from pydantic.error_wrappers import ValidationError
```

Comment:
the imports order.

Reply:
i think that you have miss this ones.

Reply:
Oh yeah I think that's because I don't run isort on `tests/` since it's less required. I'm going to manually sort it now. 

Do you think we should be running `isort` on test files as well?

Reply:
I think so

Reply:
Ok I'll make a PR for that later.